source/minio-features/erasure-coding.rst

```diff
@@ -0,0 +1,277 @@
+------------------------------
+
+MinIO uses a Reed-Solomon algorithm to split objects into data and parity blocks
+based on the size of the Erasure Set. MinIO uses parity blocks to automatically
+heal damaged or missing data blocks when reconstructing an object. MinIO uses
+the ``EC:N`` notation to refer to the number of parity blocks (``N``) in the
+Erasure Set.
+
+When applications write a new object to the MinIO cluster, MinIO randomly
+selects an Erasure Set in the cluster to store that object. For versioned
+objects, MinIO selects the same Erasure Set as all existing versions of that
+object.
+
+After MinIO selects an Erasure Set, it chunks the object for distribution in
+that set based on the number of drives in the set and the configured parity. 
+MinIO creates:
+
+- ``(Erasure Set Drives) - EC:N`` Data Blocks, *and*
+- ``EC:N`` Parity Blocks.
+
+MinIO randomly and uniformly distributes the data and parity blocks across
+drives in the erasure set with *no overlap*. While a drive may contain both data
+and parity blocks for multiple unique objects, a single unique object has no
+more than one block per drive in the set. For versioned objects, MinIO selects
+the same drives for both data and parity storage while maintaining zero overlap
+on any single drive.
+
+The specified parity for an object also dictates the minimum number of Erasure
+Set drives ("Quorum") required for MinIO to either read or write that object:
+
+Read Quorum
+   The minimum number of Erasure Set drives required for MinIO to 
+   serve read operations. MinIO can automatically reconstruct an object
+   with corrupted or missing data blocks if enough drives are online to
+   provide Read Quorum for that object.
+  
+   MinIO Read Quorum is ``DRIVES - (EC:N)``.
+
+Write Quorum
+  The minimum number of Erasure Set drives required for MinIO
+  to serve write operations. MinIO requires enough available drives to
+  eliminate the risk of split-brain scenarios. 
+  
+  MinIO Write Quorum is ``DRIVES - (EC:N-1)``.
+
+Storage Classes
+~~~~~~~~~~~~~~~
+
+MinIO supports storage classes with Erasure Coding to allow applications to
+specify per-object :ref:`parity <minio-ec-parity>`. Each storage class specifies
+a ``EC:N`` parity setting to apply to objects created with that class. 
+
+MinIO storage classes are *distinct* from Amazon Web Services storage classes.
+MinIO storage classes define *parity settings per object*, while AWS storage
+classes define *storage tiers per object*. 
+
+MinIO provides the following two storage classes:
+
+``STANDARD``
+   The ``STANDARD`` storage class is the default class for all objects. 
+
+   You can configure the ``STANDARD`` storage class parity using either:
+
+   - The :envvar:`MINIO_STORAGE_CLASS_STANDARD` environment variable, *or*
+   - The :mc:`mc admin config` command to modify the ``storage_class.standard``
+     configuration setting.
+
+   Starting with <RELEASE>, MinIO defaults ``STANDARD`` storage class to
```

Comment:
Not sure if <RELEASE> here is a variable or if you need to manually populate with the correct release number?  Calling out in case it is not a variable and needs to be input manually.

Reply:
Variable, but depends on https://github.com/minio/minio/pull/10314. That PR seems a little stale so I might revert this to N/2 and no RR by default.